_includes/layout.11ty.js

```diff
@@ -30,13 +30,19 @@ module.exports = async function(data) {
 		</ul>`;
 	}
 
+	let meta_description = `A read-only indieweb self-hosted archive of${ data.pagination && data.pagination.hrefs && data.pagination.hrefs.length ? ` all ${data.pagination.hrefs.length}` : ""} of ${data.metadata.username}’s tweets.`;
```

Comment:
Wouldn’t this duplicate "of"?

```suggestion
	let meta_description = `A read-only indieweb self-hosted archive of${ data.pagination && data.pagination.hrefs && data.pagination.hrefs.length ? ` all ${data.pagination.hrefs.length} of` : ""} ${data.metadata.username}’s tweets.`;
```

Reply:
Yes. Yes it would. Good catch.